src/main/java/bong/utils/BongException.java

```diff
@@ -0,0 +1,15 @@
+package bong.utils;
+
+/**
+ * Represents a custom exception specific to the Bong application.
+ */
+public class BongException extends Exception {
```

Comment:
It might have been better to create different subclasses for different exceptions.